src/secex-genprotimgvm-scripts/genprotimg.bu

```diff
@@ -31,6 +29,34 @@ storage:
           genprotimg -V --no-verify -i /var/genprotimg/vmlinuz -r /var/genprotimg/initrd.img  -p /var/genprotimg/parmfile -k /etc/se-hostkeys/ibm-z-hostkey-1 -o /var/genprotimg/se.img
           rm -f /var/genprotimg/signal.file
           bash /var/build/post-script.sh
+    - path: /etc/systemd/system-generators/coreos-genprotimg-generator
+      overwrite: true
+      mode: 0755
+      contents:
+        inline: |
+          #!/bin/bash
+          export PATH="/usr/bin:/usr/sbin:${PATH}"
+          set -euo pipefail
+          . /usr/lib/coreos/generator-lib.sh
+          if [ ! -z $(karg ignition.firstboot) ]; then
+              exit 0
+          fi
+          mkdir -p "${UNIT_DIR}/default.target.wants"
+
+          cat > "${UNIT_DIR}"/var-build.mount << 'EOF'
+          # generated by coreos-genprotimg-generator
+          # Mounts partition with parmfile, pre- and post- scripts used by genprotimg.service
+          [Unit]
+          Description=Genprotimg partition mount
+          Requires=dev-disk-by\x2did-virtio\x2dgenprotimg.device
+          After=dev-disk-by\x2did-virtio\x2dgenprotimg.device
+          [Mount]
+          What=/dev/disk/by-id/virtio-genprotimg
+          Where=/var/build
+          Type=ext4
+          Options=rw,noatime
+          EOF
+          ln -sf "../var-build.mount" "${UNIT_DIR}/default.target.wants/"
 systemd:
   units:
     - name: serial-getty@.service
```

Comment:
feel free to delete this if you want.. in my earlier code review comment I was just saying that if we do delete it lets put it in a separate commit with a reason even if that reason is "make it more like the ubuntu VM that is used to generate secex artifacts".